tests/integration/routes/session/put-session-test.js

```diff
@@ -264,3 +264,44 @@ test('PUT /session?include=account.profile', function (group) {
     group.end()
   })
 })
+
+test('PUT /session', function (group) {
```

Comment:
let’s rename this to something like `PUT /session with uppercase letter (hoodiehq/hoodie#499)`